doc/mgr/dashboard.rst

```diff
@@ -7,98 +7,98 @@ Overview
 --------
 
 The Ceph Dashboard is a built-in web-based Ceph management and monitoring
-application to administer various aspects and objects of the cluster. It is
-implemented as a :ref:`ceph-manager-daemon` module.
+application through which you can inspect and administer various aspects
+and resources within the cluster. It is implemented as a :ref:`ceph-manager-daemon` module.
 
 The original Ceph Dashboard that was shipped with Ceph Luminous started
-out as a simple read-only view into various run-time information and performance
-data of a Ceph cluster. It used a very simple architecture to achieve the
-original goal. However, there was a growing demand for adding more web-based
+out as a simple read-only view into run-time information and performance
+data of Ceph clusters. It used a very simple architecture to achieve the
+original goal. However, there was growing demand for richer web-based
 management capabilities, to make it easier to administer Ceph for users that
-prefer a WebUI over using the command line.
+prefer a WebUI over the CLI.
 
-The new :term:`Ceph Dashboard` module is a replacement of the previous one and
-adds a built-in web based monitoring and administration application to the Ceph
-Manager. The architecture and functionality of this new module is derived from
+The new :term:`Ceph Dashboard` module adds web based monitoring and
```

Comment:
```suggestion
The new :term:`Ceph Dashboard` module adds web-based monitoring and

Reply:
I'm finding that opinions regarding hyphens vary within the Ceph world.  I'll push this one.  Committing from the web interface makes things complicated.